### PR TITLE
Fix a default resolution bug making satpy 0.23 crash

### PR DESCRIPTION
**Pull request by mraspaud**
- [x] Tests added  
 - [x] Passes ``flake8 trollflow2``

### CODE REVIEW

File: trollflow2/tests/test_trollflow2.py [hunk 1]
```diff
@@ -557,7 +557,7 @@ def test_load_composites(self):
         scn = mock.MagicMock()
         job = {"product_list": self.product_list, "scene": scn}
         load_composites(job)
-        scn.load.assert_called_with({'ct', 'cloudtype', 'cloud_top_height'}, resolution=None, generate=False)
+        scn.load.assert_called_with({'ct', 'cloudtype', 'cloud_top_height'}, resolution='*', generate=False)
```
Comment (pnuu) [inline trollflow2/tests/test_trollflow2.py:560]: Should this have this?
```python
from trollflow2.plugins import DEFAULT

scn.load.assert_called_with({'ct', 'cloudtype', 'cloud_top_height'}, resolution=DEFAULT, generate=False)
```

Comment (mraspaud) [reply trollflow2/tests/test_trollflow2.py:560]: Indeed probavly

Comment (mraspaud) [reply trollflow2/tests/test_trollflow2.py]: Fixed

**Review by pnuu — APPROVED** [review]
LGTM. Feel free to merge if there's nothing else to be added,

**Pull request merged**